migrations/2_dev_dependencies.js

```diff
@@ -4,14 +4,20 @@ const ProxyFactory = artifacts.require("./GnosisSafeProxyFactory.sol")
 const MultiSend = artifacts.require("./MultiSend.sol")
 
 module.exports = async function (deployer, network, accounts) {
+  console.log("Migrating Batch Exchange")
+  const artefact = await migrateBatchExchange({
+    artifacts,
+    deployer,
+    network,
+    account: accounts[0],
+    web3,
+  })
+
+  const Artifactor = require("@truffle/artifactor")
+  const artifactor = new Artifactor("build/contracts/")
+  await artifactor.save(artefact)
+
```

Comment:
If you run this migration against mainnet, you would not migrate a new contract, but you would just get the original BatchExchange from the npm package, because of this line: https://github.com/gnosis/dex-contracts/blob/master/src/migration/PoC_dfusion.js#L46

Hence, I don't see the fundamental risk and I think that it is nice to have it automatically copied, and we can't forget to do the copy statement.

Reply:
Ah, I didn't see that! Then it's good as it is.